aioxmpp/ibr/service.py

```diff
@@ -0,0 +1,253 @@
+########################################################################
+# File name: service.py
+# This file is part of: aioxmpp
+#
+# LICENSE
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+########################################################################
+
+import asyncio
+
+import aioxmpp
+import logging
+
+from . import xso
+
+
+logger = logging.getLogger(__name__)
+
+
+class RegistrationService:
+    """
+    Service implementing XMPP In-Band Registration(:xep:`0077`).
+
+    This service implements the response to XMPP Pings and provides a method
```

Comment:
copy-paste typo in doc. It's not the response to XMPP Pings. Implements the In-Band Registration.